manifests/0000_70_cluster-network-operator_03_deployment-ibm-cloud-managed.yaml

```diff
@@ -94,6 +94,8 @@ spec:
           value: quay.io/openshift/origin-cluster-network-operator:latest
         - name: CLOUD_NETWORK_CONFIG_CONTROLLER_IMAGE
           value: quay.io/openshift/origin-cloud-network-config-controller:latest
+        - name: CLI_IMAGE
```

Comment:
HyperShift configures the CLI image here: https://github.com/openshift/hypershift/blob/f085b411118247a77d6cb5be15b5db4d23a15ad3/control-plane-operator/controllers/hostedcontrolplane/cno/clusternetworkoperator.go#L534

I think that it should be the same one?

Reply:
yes... uh... wait, several other controllers are already using `os.Getenv("CLI_IMAGE")` without us having defined it in CNO's environment previously... how does that work outside of hypershift?